modules/master-node-sizing.adoc

```diff
@@ -26,11 +26,23 @@ The control plane node resource requirements depend on the number of nodes in th
 
 |===
 
+On a cluster with three masters or control plane nodes, the CPU and memory usage will spike up when one of the nodes is stopped, rebooted or fails since the remaining two nodes must handle the load in order to be highly available. This is expected during the upgrades as well since the masters are cordoned, drained and rebooted serially to apply the os updates in addition to the control plane operators update. It is recommended to have the overall resource usage on the master nodes at least around half of all available capacity to handle the resource usage spikes in order to avoid cascading failures on large and dense clusters. Increase the CPU and Memory on the Master nodes accordingly.
```

Comment:
rebooted, or fails because the remaining two nodes

This is also expected during upgrades because the masters are cordoned, drained, and rebooted serially to apply the operating system updates, as well as the the control plane Operators update. To avoid cascading failures on large and dense clusters, keep the overall resource usage on the master nodes to at least half of all available capacity to handle the resource usage spikes. Increase the CPU and memory on the master nodes accordingly.

Reply:
Updated, thanks.